application/src/main/java/org/togetherjava/tjbot/commands/utils/StringDistances.java

```diff
@@ -51,6 +51,29 @@ public static <S extends CharSequence> Optional<S> autocomplete(CharSequence pre
             .min(Comparator.comparingInt(candidate -> prefixEditDistance(prefix, candidate)));
     }
 
+    /**
+     * Gives sorted suggestion to autocomplete a prefix string from the given options
+     *
+     * @param prefix the prefix to give suggestion for
+     * @param candidates all the possible suggestions
+     * @param errorMargin error margin of suggestions, error percentage / 100
+     * @return collection of autocomplete suggestions
+     */
+    public static Collection<String> autocompleteSuggestions(CharSequence prefix,
```

Comment:
Expand the existing unit tests for this. please dont leave it untested.

Reply:
not resolved. u didnt do this at all

Reply:
i did added some tests

Reply:
im sorry, i didnt see them.